R/UNIDO.R

```diff
@@ -415,17 +412,13 @@ convertUNIDO <- function(x, subtype = 'INDSTAT3', exclude = TRUE)
                 replace_NAs = 'with_USA')
     }
 
-    # process data ----
-    if ('return' == exclude)
-        return(to_exclude(madrat_mule(x), subtype))
-
     x %>%
         madrat_mule() %>%
         harmonise_column_names(subtype) %>%
         drop_na('value') %>%
         select_subsectors() %>%
         add_iso3c(subtype) %>%
-        exclude_subsectors(subtype, isTRUE(exclude)) %>%
```

Comment:
Not much point in keeping the `exclude` formal for `exclude_subsectors()` if it is never used.